src/plugins/intel_npu/src/plugin/src/compiled_model.cpp

```diff
@@ -128,17 +138,25 @@ std::shared_ptr<ov::ISyncInferRequest> CompiledModel::create_sync_infer_request(
 }
 
 void CompiledModel::export_model(std::ostream& stream) const {
-    const auto& blob = _networkPtr->compiledNetwork;
+    _logger.info("CompiledModel::export_model");
+
+    // FOr CID path mean it get the blob from compiler through pfnGetNativeBinary using
+    // networkDescription->metadata.graphHandle
+
+    // For CIP path mean it get the blob form
+    // networkDescription->compiledNetwork
```

Comment:
i think the comments are still needed here to explain the flow,
because CIP and CID paths are calling this but they are getting blob in a different way.
I keep the comments here first but fix the typos

Reply:
Maybe moving it to the definition of the functions.